conf/application.conf

```diff
@@ -142,5 +142,9 @@ contact-frontend {
   host = "http://localhost:9250"
 }
 
+business-tax-account {
```

Comment:
Rather than do this, can you add it as part of the services. Specifying the host and port. Like so:

```
      business-tax-account {
        host = localhost
        port = 9020
      }
```